intelmq/bots/outputs/files/output.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/python
```

Comment:
Can we merge the files output bot with the existing file output bot?
The similar naming might be confusing.
When to use which one of the file output bots?